src/bundle/Resources/public/scss/fieldType/_base-preview.scss

```diff
@@ -0,0 +1,116 @@
+.ez-field-edit--with-preview {
+    &,
+    *,
+    *:after,
+    *:before {
+        box-sizing: border-box;
+    }
+
+    .ez-icon {
+        fill: $ez-white;
+        stroke: $ez-white;
+        height: 1rem;
+        width: 1rem;
+    }
+
+    .form-control-file {
+        opacity: 0;
+        height: 1px;
+    }
+
+    .ez-field-edit__preview[hidden] + .ez-field-edit__data {
+        border-radius: .25rem;
+        border: 1px dashed $ez-color-primary;
+        padding: 1rem;
+        text-align: center;
+
+        .ez-data-source__btn-add {
+            display: inline-flex;
+            padding: .5rem 1.25rem;
+            border-radius: .25rem;
+            text-align: center;
+            cursor: pointer;
+            letter-spacing: .5px;
+            color: $ez-white;
+            font-size: .875rem;
+            font-weight: 700;
+            border: 1px solid $ez-color-primary;
+            background-color: $ez-color-primary;
+            transition: background .3s ease-in-out;
+            align-items: center;
+
+            .ez-icon {
```

Comment:
`_icon.scss`

Reply:
I don't agree. It's about positioning an element inside the preview. I'll keep that here.